man/systemd.xml

```diff
@@ -940,10 +940,14 @@
       <varlistentry>
         <term><varname>systemd.confirm_spawn=</varname></term>
 
-        <listitem><para>Takes a boolean argument. If
-        <option>yes</option>, the system manager (PID 1) asks for
-        confirmation when spawning processes. Defaults to
-        <option>no</option>.</para></listitem>
+        <listitem><para>Takes a boolean argument or a path to the
+        virtual console where the confirmation messages should be
+        emitted. If <option>yes</option>, the system manager (PID 1)
+        asks for confirmation when spawning processes and use by
+        default <option>/dev/console</option>. If a path or a console
+        name (such as 'ttyS0') is provided, the virtual console
+        pointed to this path or describe by the give name will be use
```

Comment:
hum... you already made me change "pointed by this path" into "pointed to this path" ;)